app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_secure_password
```

Comment:
Add a line break between the blocks. [Reference](https://compass.nimblehq.co/development/code-conventions/ruby-on-rails/#models)

Reply:
Fixed on 1a6f8a71623b59485f38ecf4ad08e901e64dfd12